minio/README.md

```diff
@@ -0,0 +1,120 @@
+# Minio
+
+A Minio storage driver using [minio/minio](https://github.com/minio/minio).
+
```

Comment:
Add a line here stating the min go version supported. You can copy it from one of the other drivers

Reply:
min go version supported added to README